### ISSUE TITLE
Error filtering

### ISSUE THREAD

**Issue by felipeZ** [nlesc-nano/flamingo#8]
The input:
```yml
smiles_file:
  data.csv
 core:
   "core.xyz"
 anchor:
   "O(C=O)[H]"
 batch_size: 1000
 filters:
   include_functional_groups:
     - "C(=O)O"
   exclude_functional_groups:
     - "S(=O)(=O)"
   scscore:
     lower_than:
       3.0
   bulkiness:
     lower_than:
       200
```

with smiles:
```csv
,smiles
0,C1C23CC45C=COC14OC2C5C=C3
1,C1C23CC45C=COC14OC2C5=CO3
2,C1C23CC45OC=CC14OC2C5C=C3
3,C1C23CC45OC=CC14OC2C5=CO3
4,C1C23C=C4C5C=CC14C1C(OC21)C35
5,C1C23CC4(C=C2)C2C5OC14C=C3C25
6,C1C23CC4(C=C2)C2C5OC14OC3C25
7,C1C23C=CC11C4OC4C(=C2)C2C3C12
8,C1C23C=CC11C=CC(C4OC24)C1=C3
9,C1C23C=CC11OC4=COC1(C=C2)C34
10,C1C23C=C4OC22OC5C3C5C14C=C2
11,C1C23C=CC4C2C4C11C=COC1=C3
12,C1C23C=CC4C2C4C11OC=CC1=C3
13,C1C23C=CC4C2OC2C3OC=CC142
14,C1C23C=CC4C2OC2C(C=CC142)=C3
15,C1C23C=CC11C4OC4C(C=C2)C1=C3
16,C1C23C=C4C=CC1(C1OC21)C1C4C31
17,C1C23C=C4C(C=CC14C1OC21)C=C3
18,C1C23C=CC(=C2)C11OC=CC1C=C3
19,C1C23C=CC11OC=CC(C=C2)C1=C3
```
Produces the following error:
```bash
<class `ValueError`> cannot call _`vectorize` on size 0 inputs unless `otypes` is set
```

**Comment by BvB93** [comment]
I suspect that adding an `if not molecules.size: return None` clause will fix the issue:
https://github.com/nlesc-nano/flamingo/blob/f9238498043685c7a4b33dd89ebd899e42647289/flamingo/screen.py#L66-L81

**Comment by felipeZ** [comment]
@BvB93 @ultimoboulevard  Is indeed that the first step  keeps only the molecules that contains a carboxylic acid but there are non. So it crashes.
I am going to add a more informative error

**Comment by felipeZ** [comment]
solved via #9
